projects/fsastorefrontstyles/scss/base/comparison-table.scss

```diff
@@ -132,4 +131,35 @@
       border-left: none;
     }
   }
+  .ribbon {
```

Comment:
Maybe place this chunk in building blocks. Perhaps we can make it into a function that's accepting some params later on so it can be reused in other places?